jetbrains-core/src/software/aws/toolkits/jetbrains/ui/SliderPanel.kt

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.ui
```

Comment:
Assume this was just a port to kotlin?

Reply:
Yep, `convert to kotlin` then go through and fixup weird things